Examples/Tests/open_bc_poisson_solver/inputs_test_3d_open_bc_poisson_solver

```diff
@@ -36,7 +36,7 @@ diagnostics.diags_names = diag1 diag2
 
 diag1.intervals = 1
 diag1.diag_type = Full
-diag1.fields_to_plot = Ex Ey Ez Bx By Bz rho
+diag1.fields_to_plot = Ex Ey Ez Bx By rho
```

Comment:
@aeriforme 

I'm removing `Bz` from the list of fields here. Just migrating from Ubuntu 20.04 to Ubuntu 24.04 produced a _relative_ error of order $O(1)$ on `Bz`. I think we discussed that the fact that `Bz` is small should not affect the relative error (precisely because it's relative), but I don't remember what other conclusion we reached on why the `Bz` values appear to be so unstable. Is it okay for you if we remove it from the fields that we consider for the checksums?

Reply:
To reconsider in #5661.